magento/magento2
# Make CSS minifying compatible with calc() CSS function

See magento/magento2#8552

### Description
Whitespace around "-" sign got removed during CSS minification due to regex done in tubalmartin/cssmin, which got fixed in https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/commit/59e5e8b0b8225244aea90da3cd5f81b1ba575da2#diff-80f11f7052c350cef68c3041a5479789R588

### Fixed Issues (if relevant)
1. magento/magento2#8552
2. magento/magento2#6822

### Manual testing scenarios
1. Set Magento in production mode (minify doesn't work on developer or default modes)
2. Extend Magento theme and use CSS calc() function in your less file. Example:
`body{width: calc((100% / 12 * 2) - 10px);}`
3. Turn Minify CSS on in admin
4. Observe results, whitespace arond "-" sign should not be removed

### Contribution checklist
